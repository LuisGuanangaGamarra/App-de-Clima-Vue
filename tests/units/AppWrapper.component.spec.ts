import { describe, it, expect } from 'vitest'
import { mount } from '@vue/test-utils'
import AppWrapper from '@/App.vue'

describe('AppWrapper', () => {
    it('renderiza el componente', () => {
        const wrapper = mount(AppWrapper)
        expect(wrapper.html()).toMatchSnapshot()
    })
})
