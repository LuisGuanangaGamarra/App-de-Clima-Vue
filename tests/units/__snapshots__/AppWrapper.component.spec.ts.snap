// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`AppWrapper > renderiza el componente 1`] = `
"<div class="min-h-screen bg-gradient-to-br from-[#cbe7ff] to-[#a3d0f7] flex flex-col items-center justify-center px-4">
  <div class="flex w-full max-w-xl items-center bg-white/80 shadow-lg rounded-full px-6 py-3 backdrop-blur-md ring-1 ring-white/20 mb-8"><input type="text" placeholder="Ingresa una ciudad..." class="flex-1 text-gray-700 text-lg bg-transparent outline-none placeholder-gray-500"><button class="cursor-pointer ml-4 bg-gradient-to-r from-blue-500 to-indigo-600 text-white font-semibold px-6 py-2 rounded-full hover:scale-105 transition-transform"> Buscar </button></div>
  <!--v-if-->
  <!--v-if-->
</div>"
`;
